File: articles/algorithms-in-labor-management-and-organizational-systems/cpp/workload_burden.cpp
#include <iostream>

int main() {
    double pace = 0.84;
    double hours = 0.72;
    double fatigue = 0.70;
    double schedule_volatility = 0.78;
    double burden = (pace + hours + fatigue + schedule_volatility) / 4.0;
    std::cout << "workload_burden_score=" << burden << "\n";
    return 0;
}

File: articles/algorithms-in-labor-management-and-organizational-systems/java/WorkloadBurden.java
public class WorkloadBurden {
    public static void main(String[] args) {
        double pace = 0.84;
        double hours = 0.72;
        double fatigue = 0.70;
        double scheduleVolatility = 0.78;
        double burden = (pace + hours + fatigue + scheduleVolatility) / 4.0;
        System.out.printf("workload_burden_score=%.4f%n", burden);
    }
}
